lib/modules/ssh/strong_password/files/ssh_listener.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+import socket
+import sys
+import threading
+import _thread
+import paramiko
+
+HOST_KEY = paramiko.RSAKey(filename='/root/.ssh/id_rsa')
+SSH_PORT = 22
+LOGFILE = '/root/logs/logins.txt'  # File to log the user:password combinations to
+LOGFILE_LOCK = threading.Lock()
+
+
+class SSHServerHandler(paramiko.ServerInterface):
+    def __init__(self):
+        self.event = threading.Event()
+
+    def check_auth_password(self, username, password):
+        LOGFILE_LOCK.acquire()
+        try:
+            logfile_handle = open(LOGFILE, "a")
+            print("New login: " + username + ":" + password)
+            logfile_handle.write(username + ":" + password + "\n")
```

Comment:
it would be the best to log IP + Country + username + password + modulename + date and time in the log.

Reply:
yes even i was thinking of this. but not sure how would i log ip and country will check this.
modulename is constant for example "ssh/strong_password" and would add it to the logs.

Reply:
great, please update the PR then, thanks.